src/test/components/__snapshots__/MarkerTooltipContents.test.js.snap

```diff
@@ -217,24 +217,24 @@ Array [
       <div
         className="tooltipLabel"
       >
-        URL
+        Status
```

Comment:
Thanks for having snapshot tests that changed! This is helpful.

Reply:
👍